worker/paidAction.js

```diff
@@ -1,9 +1,11 @@
+import { getPaymentFailureStatus, hodlInvoiceCltvDetails } from '@/api/lnd'
 import { paidActions } from '@/api/paidAction'
 import { datePivot } from '@/lib/time'
 import { Prisma } from '@prisma/client'
-import { getInvoice, settleHodlInvoice } from 'ln-service'
+import { getInvoice, getPayment, payViaPaymentRequest, settleHodlInvoice } from 'ln-service'
+import { MIN_SETTLEMENT_CLTV_DELTA } from 'wallets/wrap'
 
-async function transitionInvoice (jobName, { invoiceId, fromState, toState, toData, onTransition }, { models, lnd, boss }) {
+async function transitionInvoice (jobName, { invoiceId, fromState, toState, transition }, { models, lnd, boss }) {
```

Comment:
A lot of the changes in this file are the result of this simplification, where `transitionInvoice` callers only define `transition`.